examples/quick_start/scripts/preproc/README.md

```diff
@@ -186,6 +186,43 @@ python preprocessing.py --input_data_format=csv --csv_na_values=\\N --data_path
                         regression head for each of these targets.
 ```
 
+### Target encoding features
+
+```
+  --target_encoding_features 
+                        Columns (comma-sep) with categorical/discrete
+                        features for which target encoding features will be
+                        generated, with the average of the target columns
+                        for each categorical value. The target columns are
+                        defined in --target_encoding_targets. If
+                        --target_encoding_features is not provided but
+                        --target_encoding_targets is, all categorical
+                        features will be used.
+  --target_encoding_targets 
+                        Columns (comma-sep) with target columns that will be
```

Comment:
wont giving multiple targets create issue? you were facing issues for that.. was that fixed? also what about test set needs target column issue?

Reply:
No, I split the targets and create one TargetEncoding op for each to avoid the issue